src/System.Runtime.Serialization.Formatters/tests/BinaryFormatterTests.cs

```diff
@@ -490,8 +490,7 @@ private static void SanityCheckBlob(object obj, string[] blobs)
         {
             // These types are unstable during serialization and produce different blobs.
             if (obj is WeakReference<Point> ||
-                obj is Collections.Specialized.HybridDictionary ||
-                obj is TimeZoneInfo.AdjustmentRule)
```

Comment:
do we know why TZI.AR was added here to this test in the first place?  Was it to do with serializing test data on one machine and then re-using that on other machines (That may have different Time Zone data in the registry, etc)?

Reply:
Yes, I added it here because in the corert TimeZoneInfo sources one (option) field wasn't serialized which was in coreclr. So the blobs were different between uapaot <--> netcoreapp. Now as Jan converged the TimeZoneInfo sources into the shared folder this isn't necessary anymore :)

Reply:
Thanks